gp-templates/translation-row.php

```diff
@@ -6,248 +6,36 @@
-			<dl>
-				<dt><?php _e( 'Context:', 'glotpress' ); ?></dt>
-				<dd><span class="context bubble"><?php echo esc_translation($t->context); ?></span></dd>
-			</dl>
-			<?php endif; ?>
-			<?php if ( $t->extracted_comments ): ?>
-			<dl>
-				<dt><?php _e( 'Comment:', 'glotpress' ); ?></dt>
-				<dd><?php echo make_clickable( esc_translation($t->extracted_comments) ); ?></dd>
-			</dl>
-			<?php endif; ?>
-			<?php if ( $t->translation_added && $t->translation_added != '0000-00-00 00:00:00' ): ?>
-			<dl>
-				<dt><?php _e( 'Date added:', 'glotpress' ); ?></dt>
-				<dd><?php echo $t->translation_added; ?> GMT</dd>
-			</dl>
-			<?php endif; ?>
-			<?php if ( $t->user ) : ?>
-			<dl>
-				<dt><?php _e( 'Translated by:', 'glotpress' ); ?></dt>
-				<dd><?php gp_link_user( $t->user ); ?></dd>
-			</dl>
-			<?php endif; ?>
-			<?php if ( $t->user_last_modified && ( ! $t->user || $t->user->ID !== $t->user_last_modified->ID ) ) : ?>
-				<dl>
-					<dt><?php
-						if ( 'current' === $t->translation_status ) {
-							_e( 'Approved by:', 'glotpress' );
-						} elseif ( 'rejected' === $t->translation_status ) {
-							_e( 'Rejected by:', 'glotpress' );
-						} else {
-							_e( 'Last updated by:', 'glotpress' );
-						}
-						?>
-					</dt>
-					<dd><?php gp_link_user( $t->user_last_modified ); ?></dd>
-				</dl>
-			<?php endif; ?>
-			<?php references( $project, $t ); ?>
-
-			<dl>
-			    <dt><?php _e( 'Priority:', 'glotpress' ); ?></dt>
-			<?php if ( $can_write ): ?>
-				<dd><?php
-					echo gp_select(
-						'priority-' . $t->original_id,
-						GP::$original->get_static( 'priorities' ),
-						$t->priority,
-						array(
-							'class'      => 'priority',
-							'tabindex'   => '-1',
-							'data-nonce' => wp_create_nonce( 'set-priority_' . $t->original_id ),
-						)
-					);
-					?></dd>
-			<?php else : ?>
-				<dd><?php echo gp_array_get( GP::$original->get_static( 'priorities' ), $t->priority, 'unknown' ); // WPCS: XSS ok. ?></dd>
-			<?php endif; ?>
-			</dl>
-
-			<dl>
-			    <dt><?php _e( 'More links:', 'glotpress' ); ?>
-				<ul>
-					<?php foreach ( $more_links as $link ) : ?>
-						<li><?php echo $link; // WPCS: XSS ok. ?></li>
-					<?php endforeach; ?>
-				</ul>
-				</dt>
-			</dl>
-		</div>
-	</td>
-</tr>
+gp_tmpl_load(
+	'translation-row-preview', array(
+		'translation'             => $translation,
+		'can_edit'                => $can_edit,
+		'can_approve'             => $can_approve,
+		'can_approve_translation' => $can_approve_translation,
+	)
+);
+gp_tmpl_load(
+	'translation-row-editor', array(
+		'translation'             => $translation,
+		'can_write'               => $can_write,
+		'can_edit'                => $can_edit,
+		'can_approve'             => $can_approve,
+		'can_approve_translation' => $can_approve_translation,
+		'can_edit'                => $can_edit,
+		'locale'                  => $locale,
+		'project'                 => $project,
+		'translation_set'         => $translation_set,
+	)
```

Comment:
I'd suggest passing in all of the variables that were available in the old unified template here, in case a plugin was expecting them.

Reply:
The old template was using `get_defined_vars()` which seems like a bad idea so I changed it. Since we are targeting a major release I think it would be fine to explicitly pass the vars. We could also document the passed vars at the top of the template using @var. I see now that  `get_defined_vars()` is used in the other templates so I should change there too.

Reply:
`get_defined_vars()` is used throughout GP in this manner, and while not ideal, I think maintaining consistency with the rest of the code base is probably better than changing it in just one location.

Perhaps for 4.0 we could look at changing this for all of GP.